import { AppError } from './appError';

export default class NotFoundError extends AppError {
  constructor(message = 'Recurso n√£o encontrado') {
    super(404, message);
    this.name = 'NotFoundError';
  }
}
